front-end/src/components/LeftPane/Users/LeftPane.tsx

```diff
@@ -11,28 +11,46 @@ export interface ILeftPaneProps {
 
 export default function LeftPane(props: ILeftPaneProps) {
   const ctx = useContext(messageContext)
+  const currentUser = localStorage.getItem('username')
   const history = useHistory()
   const leaveRoom = async () => {
-    console.log('leaving')
     const userLeft = await removeFromRoom(props.roomId)
     if (userLeft) {
       history.push('/')
     }
   }
+  useEffect(() => {
+    const getRoomDetails = async () => {
+      let { roomOwnerUsername } = await getRoomInfo(props.roomId)
+      setRoomOwnerUsername(roomOwnerUsername)
+    }
+    getRoomDetails()
+  }, [props.roomId])
+  const [roomOwnerUsername, setRoomOwnerUsername] = useState('')
   return (
     <div className={styles['left-pane']}>
       <h1 className={styles['user-list-heading']}>Users</h1>
       <div className={styles['user-list-users']}>
-        {ctx.users.map((user) => (
-          <li key={Math.random()}>
-            <img src={props.image} alt='' className={styles['user-image']} />
-            {user.username}
-          </li>
-        ))}
-      </div>
-      <div className={styles['exit-group']}>
-        <Button text='Leave Room' onClick={leaveRoom} color='red' />
+        <li key={Math.random()}>
+          <img src={props.image} alt='' className={styles['user-image']} />
+          {roomOwnerUsername} 👑
+        </li>
+        {ctx.users
+          .filter((user) => user.username !== roomOwnerUsername)
+          .map((user) => (
+            <li key={Math.random()}>
+              <img src={props.image} alt='' className={styles['user-image']} />
+              {user.username}
+            </li>
+          ))}
       </div>
+      {currentUser !== roomOwnerUsername ? (
+        <div className={styles['exit-group']}>
+          <Button text='Leave Room' onClick={leaveRoom} color='red' />
+        </div>
+      ) : (
+        ''
+      )}
```

Comment:
You can replace the ternary with `&&` since nothing is being rendered in the else part anyway